### PR TITLE
[#1346] improvement : Add badges

### PR DESCRIPTION
**Pull request by hiirrxnn**
What changes were proposed in this pull request?
 Add badges to README file

Why are the changes needed?
 Fix: #1346 

Does this PR introduce any user-facing change?
 No

How was this patch tested?
 No need.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,8 +95,10 @@ Gravitino provides a Trino connector to access the metadata in Gravitino. To use
 2. [How to test Gravitino](docs/how-to-test.md)
 3. [How to publish Docker images](docs/publish-docker-images.md)
 
+
```
Comment (qqqttt123) [inline README.md:98]: Why do we need an extra blank line here?

File: README.md [hunk 2]
```diff
@@ -90,8 +95,10 @@ Gravitino provides a Trino connector to access the metadata in Gravitino. To use
...
 ## License
 
 Gravitino is under the Apache License Version 2.0, See the [LICENSE](LICENSE) for the details.
 
 <sub>Apache®, Apache Hadoop&reg;, Apache Hive&trade;, Apache Iceberg&trade;, Apache Kafka&reg;, Apache Spark&trade;, Apache Submarine&trade;, Apache Thrift&trade; and Apache Zeppelin&trade; are either registered trademarks or trademarks of the Apache Software Foundation in the United States and/or other countries.</sub>
+
```
Comment (qqqttt123) [inline README.md:104]: Why do we need an extra blank line here?

**Comment by qqqttt123** [comment]
Could you correct your description and title referring to https://github.com/datastrato/gravitino/pull/1353 ?

File: README.md (hunk 2 above)
Comment (hiirrxnn) [reply README.md]: Must have accidentally added them while scrolling through the doc , I hope it's fine now!

**Review by qqqttt123 — APPROVED** [review]
LGTM.

**Comment by qqqttt123** [comment]
Thanks @hiirrxnn , merged to master.

**Pull request merged**